docs/api_requests/block-io-engine.md

```diff
@@ -51,6 +51,56 @@ If a block device is configured with the `Async` io_engine on a host kernel
 older than 5.10.51, the API call will return a 400 Bad Request, with a
 suggestive error message.
 
+## Performance considerations
+
+An important thing to note is that the performance is strictly tied to the host
+kernel version. The gathered data may not be relevant for modified/newer
+kernels than 5.10.
+
+### Device creation
+
+When using the `Async` variant, there is added latency on device creation (up
+to ~110 ms), caused by the extra io_uring system calls performed by
+Firecracker.
+This translates to higher latencies on either of these operations:
+
+- API call duration for block device config
+- Boot time for VMs started via JSON config files
+- Snapshot restore time
+
+For use-cases where the lowest latency on the aforementioned operations is
+desired, it is recommended to use the `Sync` IO engine.
+
+### Block IOPS and efficiency
```

Comment:
I would also mention that when having multiple VMs it's recommended to distribute the backing files equally among all the NVMEs available on the host for better performance.

Reply:
done